src/api/app/models/scm_status_report.rb

```diff
@@ -0,0 +1,43 @@
+# This class represents the communication that took place between OBS and the
+# SCM to render each one of the check marks.
+#
+# Those check marks appear as green, orange and red ticks at the bottom of a
+# PR or next to a commit, on GitHub.
+#
+# The details of the request and the response will be shown in the
+# Reports to the SCM tab in Workflow Runs show page.
+class ScmStatusReport < ApplicationRecord
```

Comment:
```suggestion
class SCMStatusReport < ApplicationRecord
```

Reply:
just realized we write it upercase everywhere else, you need to set this in the `zeitwerk.rb` file